### PR TITLE
Fix #190 - Add ability to run a job right now.

### LINKED ISSUE #190: Scheduled jobs should have an option to be run "right now"
https://bugzilla.mozilla.org/show_bug.cgi?id=1309567

### PR DESCRIPTION
**Pull request by jezdez**

### CODE REVIEW

**Comment by mreid-moz** [comment]
Just driving by to say 👍

File: atmo/templates/atmo/jobs/run.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+{% block page_title %}Run Spark job {{ spark_job }}{% endblock %}
+
+{% block content %}
+<div class="page-header">
+  <h2>Run Spark job <small>{{ spark_job }}</small></h2>
+</div>
+<div class="row">
+  <div class="col-sm-6">
+    <h4>Do you really want to run the Spark ?</h4>
```
Comment (robhudson) [inline atmo/templates/atmo/jobs/run.html]: I want to run the spark! This sounds like a sci-fi movie I'm forgetting.

File: atmo/templates/atmo/jobs/delete.html [hunk 2]
```diff
@@ -9,10 +9,7 @@
 </div>
 <div class="row">
   <div class="col-sm-6">
-    <h4>Do you really want to terminate the cluster?</h4>
```
Comment (robhudson) [inline atmo/templates/atmo/jobs/delete.html]: Good catch.

File: atmo/jobs/views.py [hunk 3]
```diff
@@ -133,3 +136,42 @@ def download_spark_job(request, id):
     )
     response['Content-Length'] = spark_job.notebook_s3_object['ContentLength']
     return response
+
+
+@login_required
+@view_permission_required(SparkJob)
+def run_spark_job(request, id):
+    spark_job = SparkJob.objects.get(pk=id)
```
Comment (robhudson) [inline atmo/jobs/views.py]: Should this be a get or 404?

Comment (jezdez) [inline atmo/jobs/views.py]: yeah, it should, good catch.

File: atmo/templates/atmo/jobs/run.html (hunk 1 above)
Comment (jezdez) [inline atmo/templates/atmo/jobs/run.html]: LOL

File: atmo/jobs/views.py (hunk 3 above)
Comment (jezdez) [inline atmo/jobs/views.py]: Ah, wait, the 404 is already handled by the view_permission_required decorator, which is why there is no need to do that. I should probably add an option to make it clear that this is what is happening.

**Comment by jezdez** [comment]
The job detail page will look like this:
screen-shot-2017-04-27-15-17-41

**Pull request merged**